pydanticrud/backends/dynamodb.py

```diff
@@ -409,3 +412,30 @@ def save(self, item, condition: Optional[Rule] = None) -> bool:
 
     def delete(self, key):
         self.get_table().delete_item(Key=self._key_param_to_dict(key))
+
+    def batch_save(self, items: list) -> dict:
+        """
+        This function is to write multiple records in to dynamodb and returns unprocessed records in dict
+        if something gone wrong with the record.Currently, batch_write is not supporting ConditionExpression
+        Refer docs:
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/batch_write_item.html
+        """
+        # Prepare the batch write requests
+        request_items = {self.table_name: []}
+
+        # chunk list for size limit of 25 items to write using this batch_write operation refer below.
+        # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/batch_write_item.html#:~:text=The%20BatchWriteItem%20operation,Data%20Types.
+        for chunk in chunk_list(items, 25):
```

Comment:
This isn't a problem for notifications since we won't hit 16mb but could be a configuration.

Reply:
ok